qemu/tests/migration_multi_host.py

```diff
@@ -31,7 +31,7 @@ def run(test, params, env):
     class TestMultihostMigration(mig_type):
 
         def __init__(self, test, params, env):
-            super(TestMultihostMigration, self).__init__(test, params, env)
+            super(TestMultihostMigration, self).__init__(test, params, env, False)
```

Comment:
OK. Thanks for explain. But seems it is better to make it configurable but not hard code here.